### ISSUE TITLE
The `max_table_width:` argument to `pack` is ignored if title is too long

### ISSUE THREAD

**Issue by matt-harvey** [matt-harvey/tabulo#20]
If the table has a title, and it's longer than the argument passed to `.pack(max_table_width: <here>)` (or than the screen width if `:auto` is passed or if the argument is omitted), then the table will expand beyond the passed width. I don't think this is the behaviour that users of the library would expect; and it's not the documented behaviour. So I would characterise this as a bug. 

To reproduce this:

```rb
require 'ostruct'
require 'tabulo'

data = [OpenStruct.new(foo: 'foovalue', bar: 'barvalue')]
table = Tabulo::Table.new(data, :foo, :bar, title: 'abcdefghijklmnopqrstuvwxyz 1234567890')
puts table.pack(max_table_width: 20)
```

**Comment by matt-harvey** [comment]
Fixed by 2a028e0986c16809f295612637f23cf41acd8a2e.

Will release in v2.8.

**Comment by matt-harvey** [comment]
Fix has been released in v2.8.0.
